Makefile

```diff
@@ -861,7 +861,7 @@ pkgs: build-tools $(PKGS)
 pkg/kernel:
 	$(QUIET): $@: No-op pkg/kernel
 
-pkg/external-boot-image/build.yml: pkg/external-boot-image/build.yml.in
```

Comment:
Would it make sense to also comment the use of FORCE here in the Makefile itself?

Reply:
Makes sense, it's done.